### PR TITLE
Bbstilson/update documentation for boto3

### PR DESCRIPTION
**Pull request by bbstilson**
Noop change. Just fixes documentation issues here:  https://github.com/aio-libs/aiobotocore/issues/997

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -9,18 +9,18 @@ Running Tests
 Thanks for your interest in contributing to ``aiobotocore``, there are multiple
 ways and places you can contribute.
 
-Fist of all just clone repository::
```
Comment (thehesiod) [inline CONTRIBUTING.rst:12]: 🤣

**Review by thehesiod — APPROVED** [review]

**Comment by thehesiod** [comment]
ty!

**Comment by thehesiod** [comment]
@bbstilson looks like you have a flake error

**Comment by bbstilson** [comment]
This should be ready for another CI run @thehesiod

**Pull request merged**